mmv1/products/cloudfunctions2/api.yaml

```diff
@@ -274,29 +274,50 @@ objects:
           An Eventarc trigger managed by Google Cloud Functions that fires events in
           response to a condition in another service.
         properties:
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'trigger'
             description: 'The resource name of the Eventarc trigger.'
             output: true
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'triggerRegion'
             description: |
               The region that the trigger will be in. The trigger will only receive
               events originating in this region. It can be the same
               region as the function, a different region or multi-region, or the global
               region. If not provided, defaults to the same region as the function.
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::String
             name: 'eventType'
             description: 'Required. The type of event to observe.'
-          - !ruby/object:Api::Type::String 
+          - !ruby/object:Api::Type::Array
+            name: 'eventFilters'
+            description: 'Criteria used to filter events.'
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: attribute
+                  description: |
+                    'Required. The name of a CloudEvents attribute.
+                    Currently, only a subset of attributes are supported for filtering. Use the `gcloud eventarc providers describe` command to learn more about events and their attributes.
+                    Do not filter for the 'type' attribute here, as this is already achieved by the resource's `event_type` attribute.
+                - !ruby/object:Api::Type::String
+                  name: value
+                  description: 'Required. The value for the attribute.'
+                - !ruby/object:Api::Type::String
+                  name: operator
```

Comment:
Ok... good news is that I've found out how this `operator` field is used ([blog post](https://atamel.dev/posts/2022/03-02_path_patterns_eventarc/), docs on [path patterns](https://cloud.google.com/eventarc/docs/path-patterns)). I didn't know Eventarc was used for cloundfunction2 when I started the ticket, so I overlooked stuff in that service's docs that can help with this, and then I got sucked into the acceptance test wormhole.

The bad news is that usage of the `operator` field is [not supported for every filterable event attribute](https://cloud.google.com/eventarc/docs/path-patterns), and none of the GCS events' attributes are filterable, so this would require a new test example 😅

Reply:
Luckily replicating the example in that blog post & in the docs was easy to do - example of `operator` being used is in 
a5b4228